### PR TITLE
Autopilot code linter fix

### PR DESCRIPTION
**Pull request by makhov**
## Description

Fixes autopilot linter issues.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## How Has This Been Tested?

- [ ] Manual test
- [ ] Auto test added

## Checklist:

- [ ] My code follows the style [guidelines](https://github.com/k0sproject/k0s/blob/main/docs/contributors/overview.md) of this project 
- [ ] My commit messages are [signed-off](https://github.com/k0sproject/k0s/blob/main/docs/contributors/github_workflow.md)
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

File: .github/workflows/lint.yaml [hunk 1]
```diff
@@ -57,10 +57,8 @@ jobs:
...
         run: make codegen EMBEDDED_BINS_BUILDMODE=none && git diff --exit-code
 
-      - name: golangci-lint
-        uses: golangci/golangci-lint-action@v3
-        with:
-          version: 'v${{ env.GOLANGCI_LINT_VERSION }}'
-          skip-cache: true
-          only-new-issues: false
-          args: --verbose
+      - name: Run linter
```
Comment (s0j) [inline .github/workflows/lint.yaml:60]: I don't think changing how the CI lint works should be a part of the autopilot integration.

**Review by s0j — CHANGES_REQUESTED** [review]

File: go.mod [hunk 2]
```diff
@@ -48,17 +53,12 @@ require (
...
-	sigs.k8s.io/yaml v1.3.0
-	github.com/Masterminds/semver/v3 v3.1.1
-	github.com/cavaliergopher/grab/v3 v3.0.1
-	github.com/go-playground/validator/v10 v10.9.0
-	github.com/google/go-cmp v0.5.7
-	github.com/robfig/cron v1.2.0
 	gopkg.in/yaml.v2 v2.4.0
+	helm.sh/helm/v3 v3.8.2
 	k8s.io/apiextensions-apiserver v0.24.0
 	k8s.io/kubernetes v1.13.0
```
[the remark this answers is not in the document]
Comment (makhov) [reply go.mod]: Sorry, did it temporarily and forgot to update. Fixed now

**Review by s0j — APPROVED** [review]
I'm fine with this once the redunant manifest directory is removed, or if that is done in another PR.

**Comment by makhov** [comment]
Thanks! I'll remove the redundant dir in a separate PR.

**Pull request merged**